<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_mas">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Count Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_menos">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Count Down </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_reset">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="text_contador">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>66</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Contador:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>539</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
